src/subcommand/giraffe_main.cpp

```diff
@@ -1776,8 +1808,19 @@ int main_giraffe(int argc, char** argv) {
         cerr << "Loading Minimizer Index" << endl;
     }
     unique_ptr<gbwtgraph::DefaultMinimizerIndex> minimizer_index;
+    unique_ptr<gbwtgraph::MinimizerIndexXL> path_minimizer_index(new gbwtgraph::MinimizerIndexXL());
```

Comment:
I don't think we want to have this non-null but pointing at an empty index here. We should only construct the index if we're really loading it.